pkg/services/sqlstore/migrations/alert_mig.go

```diff
@@ -71,6 +71,9 @@ func addAlertMigrations(mg *Migrator) {
 	mg.AddMigration("Add column send_reminder", NewAddColumnMigration(alert_notification, &Column{
 		Name: "send_reminder", Type: DB_Bool, Nullable: true, Default: "0",
 	}))
+	mg.AddMigration("Add column disable_resolved_message", NewAddColumnMigration(alert_notification, &Column{
+		Name: "disable_resolved_message", Type: DB_Bool, Nullable: false, Default: "1",
```

Comment:
This should default to `0` otherwise we will introduce a breaking behavior for those who expect the `Ok` notification.

Reply:
Thanks for the catch.  Fixed, and added test for this as well.